### ISSUE TITLE
API throws "internal_api_error_MWException" on MW 1.25

### ISSUE THREAD

**Issue by mwjames** [SemanticMediaWiki/SummaryCards#3]
I only tested this on 1.28 and the sandbox complains about 

```
[85199326] Exception Caught: Internal error in ApiBase::getParameterFromSettings: Param text's type is unknown - text

"#0 /var/www/htdocs/mw/02100/w/includes/api/ApiBase.php(946): ApiBase::dieDebug(string, string)#1 /var/www/htdocs/mw/02100/w/includes/api/ApiBase.php(564): ApiBase->getParameterFromSettings(string, array, boolean)#2 /var/www/htdocs/mw/02100/w/includes/api/ApiMain.php(968): ApiBase->extractRequestParams()#3 /var/www/htdocs/mw/02100/w/includes/api/ApiMain.php(1117): ApiMain->setupModule()#4 /var/www/htdocs/mw/02100/w/includes/api/ApiMain.php(469): ApiMain->executeAction()#5 /var/www/htdocs/mw/02100/w/includes/api/ApiMain.php(442): ApiMain->executeActionWithErrorHandling()#6 /var/www/htdocs/mw/02100/w/api.php(88): ApiMain->execute()#7 {main}"
```

**Comment by mwjames** [comment]
@kghbln Is the reason why we currently only see:

![image](https://cloud.githubusercontent.com/assets/1245473/15748267/82b3ac60-28de-11e6-9042-77be4525d42e.png)

**Comment by kghbln** [comment]
Affirmative. Saw this on "Special:Version". Probably another idea will be to exclude this from working in namespace "Special".

**Comment by mwjames** [comment]
Link filtering is a bit of magic since there  are no clear rules in MW on how to filter a href. Current rules are [0].

[0] https://github.com/SemanticMediaWiki/SummaryCards/blob/master/res/suc.cards.js#L93-L135

**Comment by mwjames** [comment]
> Saw this on "Special:Version". Probably another idea will be to exclude this from working in namespace "Special".

I fixed this in one go (with #4). If we find links we see unfit then hopefully we can define some rule set to get them not selected in the first place.

**Comment by kghbln** [comment]
> I fixed this in one go (with #4). 

Cool, now live on sandbox. Need a cuppa tea now :)
